Net/src/TCPServerDispatcher.cpp

```diff
@@ -96,7 +96,7 @@ void TCPServerDispatcher::release()
 
 void TCPServerDispatcher::run()
 {
-	AutoPtr<TCPServerDispatcher> guard(this, true); // ensure object stays alive
+	AutoPtr<TCPServerDispatcher> guard(this, false); // ensure _rc is decreased when function exits
```

Comment:
false does nothing here, so it can be just `guard(this)`. It would be good to point to the relationship of this with enqueue `++_rc`, since it's a functionality in two distant places

Reply:
Updated, thanks.